Lib/tkinter/test/test_tkinter/test_misc.py

```diff
@@ -98,6 +98,17 @@ def callback(start=0, step=1):
         with self.assertRaises(tkinter.TclError):
             root.tk.call(script)
 
+        # Call with a callable class
```

Comment:
It looks unnecessary complex. Would not `partial()` be enough?

Reply:
done